src/main/proto/wfa/measurement/reporting/v1alpha/report_parameter.proto

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package src.main.proto.wfa.measurement.reporting.v1alpha;
+
+option java_package = "com.src.main.proto.wfa.measurement.reporting.v1alpha";
```

Comment:
@SanjayVas I modified `package` and `java_package` based on API Linter's suggestion, but it's different from the format we have in other .proto. Will that be an issue?